connector/src/main/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptions.scala

```diff
@@ -146,6 +148,23 @@ object SparkBigQueryOptions {
           .stripMargin.replace('\n', ' '))
     }
     val readDataFormat = DataFormat.valueOf(readDataFormatParam)
+
+    val arrowEnableUnsafeMemoryAccessProperty = "arrow.enable_unsafe_memory_access"
```

Comment:
Until we unify SpringBigQueryOptions and SparkBIgQueryConfig, please implement the setting in both places, so that the DataSource v2 implementation will have this as well

Reply:
Added corresponding changes to SparkBigQueryConfig and SparkBigQueryConfigSuite. Is that enough?